workspace-cache-server/README.md

```diff
@@ -0,0 +1,43 @@
+# workspace-cache-server
+
+> :herb: This is the cache server workspaces :octocat: All you need in a nut:shell:
+
+This template is a simple express API using a postgresql database. To run this example you need a postgresql database and create the tables. The tables for this example are at *start.sql* file.
+
+In case you want or need to interact with the database, [here](https://wiki.postgresql.org/wiki/PostgreSQL_Clients) is a full list of clients.
+
+This basic example is written is typescript.
+
+## Installation
+
+Use [yarn](https://yarnpkg.com) to install dependencies.
+
+```bash
+$ yarn
+```
+
+## Usage
+
+### Start a database
+
+To keep everything simple, we recommend you to start  postgresql database, using [docker](https://www.docker.com/). If you already have, follow the instructions [here](https://hub.docker.com/_/postgres) to start a local postgresql database.
+
```

Comment:
I couldn't follow those instructions, they are just not copy & paste friendly. This one worked for me: https://medium.com/better-programming/connect-from-local-machine-to-postgresql-docker-container-f785f00461a7

Reply:
They are. I use `docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres` and then connect using [postbird](https://www.electronjs.org/apps/postbird), with
```
Host: localhost
Port: 5432
Username: postgres
Password: mysecretpassword
```
You should be able to do the same.

Reply:
Then be just a nice guy to your readers, and paste those instructions in the README :)